requirements/base.txt

```diff
@@ -5,7 +5,7 @@
 #
 #    pip-compile-multi
 #
-attrs==22.2.0 \
-    --hash=sha256:29e95c7f6778868dbd49170f98f8818f78f3dc5e0e37c0b1f474e3561b240836 \
-    --hash=sha256:c9227bfc2f01993c03f68db37d1d15c9690188323c067c641f1a35ca58185f99
+attrs==23.2.0 \
+    --hash=sha256:935dc3b529c262f6cf76e50877d35a4bd3c1de194fd41f47a2b7ae8f19971f30 \
+    --hash=sha256:99b87a485a5820b23b879f04c2305b44b951b502fd64be915879d77a7e8fc6f1
```

Comment:
Unrelated to this patch, but I think we should migrate from attrs -> dataclasses wherever possible. In my experience we never use any of the advanced features of attrs that differentiate it from dataclasses.

Reply:
Oh, I remember dataclasses weren't powerful enough when I first wrote mozilla-version. I don't remember the specifics. Maybe it's worth reevaluating nowadays! 🙂